docker/web/Dockerfile

```
@@ -1,9 +1,10 @@
-ARG CADDY_VERSION="2.9.1"
+ARG CADDY_VERSION="2.10.0"
+ARG S3_CADDY_VERSION="0.10.0"
 
 FROM caddy:$CADDY_VERSION-builder-alpine AS builder
 
 RUN xcaddy build \
-    --with github.com/sagikazarmark/caddy-fs-s3
+    --with github.com/sagikazarmark/caddy-fs-s3@v0.10.0
```

Looks like you intended to use the `S3_CADDY_VERSION` variable here

```suggestion
    --with github.com/sagikazarmark/caddy-fs-s3@$S3_CADDY_VERSION
```